store/src/test/java/in/testpress/store/network/ProductListNetworkTest.kt

```diff
@@ -0,0 +1,28 @@
+package `in`.testpress.store.network
+
+import kotlinx.coroutines.runBlocking
+import okhttp3.mockwebserver.MockResponse
+import org.junit.*
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class ProductListNetworkTest: NetworkTestMixin() {
+
+    @Test
+    fun networkCallShouldReturnFetchedData() {
```

Comment:
This name is generic. In most cases, network calls will return some data. You should test whether you are getting the correct ProductListResponse or not.